cmake/compiler/arcmwdt/compiler_flags.cmake

```diff
@@ -168,6 +168,8 @@ set_compiler_property(PROPERTY imacros -imacros)
 # Security canaries.
 #no support of -mstack-protector-guard=global"
 set_compiler_property(PROPERTY security_canaries -fstack-protector-all)
+set_compiler_property(PROPERTY security_canaries_strong -fstack-protector-strong)
+set_compiler_property(PROPERTY security_canaries_explicit -fstack-protector-explicit)
```

Comment:
MetaWare is like llvm/clang, there is no "-fstack-protector-explicit" option, and instead of `__attribute__((stack_protector))`  the ` __attribute__((no_stack_protector))` is used for explicit control.

Reply:
So it is the other way around, instead of tells which functions the compiler should instrument you tells which ones shouldn't ?